docs/en/ingest-management/integrations/upgrade-integration.asciidoc

```diff
@@ -0,0 +1,147 @@
+*Upgrade integration policies*. You can still choose to
+<<upgrade-integration-policies-manually,upgrade integration policies manually>>
+later on.
+--
+
+. Click *Upgrade to latest version*.
++
+If you selected *Upgrade integration policies* and there are conflicts,
+<<upgrade-integration-policies-manually,upgrade integration policies manually>>
+and resolve the conflicts in the policy editor.
+
+. After the upgrade is complete, verify that the installed version and latest
+version match.
+
+[discrete]
+[[upgrade-integration-policies-automatically]]
+== Keep integration policies up to date automatically
+
+Some internal integration packages, like System, are upgraded automatically when
+{fleet} detects that a new version is available. These integrations have an
+option to upgrade integration policies automatically, too. When this option is
+selected (the default), {fleet} automatically upgrades your policies behind the
+scenes when a new version of the integration is available. If there are
+conflicts during the upgrade, your integration policies will not be upgraded,
+and you'll need to
+<<upgrade-integration-policies-manually,upgrade integration policies manually>>.
+
+//QUESTION: Is it true that this setting on by default for all internal integrations? 
+
+To keep integration policies up to data automatically:
+
+. In {kib}, go to *Management > Integrations > Installed*. Search for and select
+the integration you want to configure.
+
+. Click *Settings* and make sure
+*Keep integration policies up to data automatically* is selected. 
++
+[role="screenshot"]
+image::images/upgrade-integration-policies-automatically.png[Settings tab under Integrations shows how to keep integration policies up to date automatically]
++
+If this option isn't available on the *Settings* tab, this feature is not
+available for the integration you're viewing.
+
+
+[discrete]
+[[upgrade-integration-policies-manually]]
+== Upgrade integration policies manually
+
+If you can't upgrade integration policies when you upgrade the integration,
+upgrade them manually.
+
+. Click the *Policies* tab and find the integration policies you want to
+upgrade.
++
+[role="screenshot"]
+image::images/upgrade-package-policy.png[Policies tab under Integrations shows how to upgrade the package policy]
+
+. Click *Upgrade* to begin the upgrade process.
++
+The upgrade will open in the policy editor.
++
+[role="screenshot"]
+image::images/upgrade-policy-editor.png[Upgrade integration example in the policy editor]
+
+. Make any required configuration changes and, if necessary, resolve conflicts.
+For more information, refer to <<resolve-conflicts>>.
+
+. Repeat this process for each policy with an out-of-date integration.
+
+Too many conflicts to resolve? Refer to the 
+<<upgrading-integration-too-many-conflicts,troubleshooting docs>> for manual
+steps.
+
+//QUESTION: Do we still want to recommend the troubleshooting steps shown here?
```

Comment:
I think this still seems appropriate to me.  The older method of recreating policies from scratch is still a viable escape hatch for problematic upgrades.